.gitmodules

```diff
@@ -32,3 +32,5 @@
 [submodule "suru-plus"]
 	path = plugins/icons_suruplus/suru-plus
 	url = https://github.com/gusbemacbe/suru-plus.git
+
+
```

Comment:
it's working not like that :-)

Reply:
1) fork oomox

2) install your oomox fork manually: https://github.com/themix-project/oomox#installation-1

3) go to subdirectory with Suru repo, and update Suru to the desired version there

4) go back to oomox fork repo root and make commit for submodule update

5) make a pull request

Reply:
> 3.go to subdirectory with Suru repo, and update Suru to the desired version there

```
cd plugins/icons_suruplus/suru-plus
git checkout master
git pull
cd ..
git commit -am "Updated Suru++ submodule"
git push
```

Did I make something wrong?

Reply:
usually it's safer  to do `git pull origin master` instead of just `git pull`

Reply:
(and the same for push)

Reply:
too see the correct name (if it's not `origin`) see `git remote -v`

Reply:
Is it normal for this error?

![imagem](https://user-images.githubusercontent.com/2257024/53801555-4f173300-3f1e-11e9-8a70-419cb6d0abd6.png)

Reply:
if you cloned your fork repo as described here (https://github.com/themix-project/oomox#installation-1) it shouldn't happen

Reply:
Ah, I mixed the branches. I had to remove again and to clone again and will be careful.

Reply:
Ready, @actionless !

Reply:
thanks, now rebase on top of master in order to remove you previous mistaken commit (which adds two blank lines)

or, alternatively, add new commit which removing those lines anbd i'll squash all PR into one commit during merging

Reply:
Ready, I hope I did something right.

Reply:
i don't see anything happened

you can read up here how to delete some particular commit while doing interactive rebasing on top of master (`git rebase -i master`):
https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d

Reply:
Too bad I failed. I have rebased, thinking I removed the commit, but the commit has not been removed. Sorry! I have to redo everything.

Reply:
no problem, take your time

understanding git rebase and doing it for the first time could be difficult

Reply:
though when doing it second time you will be surprised how hard all your previous life was without knowing it :D

Reply:
I could choose the option 2:

> or, alternatively, add new commit which removing those lines anbd i'll squash all PR into one commit during merging

Because it is not a nuclear bomb. I chose the option 1, and hit fire and an atomic bomb in my repository.